src/main/scala/mesosphere/marathon/api/akkahttp/v2/PodsController.scala

```diff
@@ -171,7 +174,19 @@ class PodsController(
 
   def version(podId: PathId, v: String): Route = ???
 
-  def allStatus(): Route = ???
+  def allStatus(): Route =
+    authenticated.apply { implicit identity =>
+      def isAuthorized(pod: PodDefinition): Boolean = authorizer.isAuthorized(identity, ViewRunSpec, pod)
+
+      val filteredPods = Source(podManager.ids())
+        .mapAsync(Int.MaxValue) { id =>
```

Comment:
Do you think it's worth limiting amount of concurrent calls here?

Reply:
I'd say so but we don't have a common way to configure this yet. So I'd rather follow up.

Reply:
Well, let's just hardcode the limit for now :)